client/signup/steps/about/style.scss

```diff
@@ -95,6 +95,13 @@
 .about__submit-wrapper {
 	text-align: center;
 	margin-bottom: 20px;
+	padding: 0 15px;
+
+	.button {
```

Comment:
I think we may not need to nest this since there's also just one selector (`.button`) under `.about__submit-wrapper`.